frontend/src/util/MessfaehigkeitRanking.ts

```diff
@@ -0,0 +1,42 @@
+import Fahrzeugklasse from "@/types/enum/Fahrzeugklasse";
+import ZaehldatenIntervall from "@/types/enum/ZaehldatenIntervall";
+
+export function useMessfaehigkeitRanking() {
+
+    let rankingFahrzeugklassen: Fahrzeugklasse[] = [Fahrzeugklasse.SUMME_KFZ, Fahrzeugklasse.ZWEI_PLUS_EINS, Fahrzeugklasse.ACHT_PLUS_EINS];
+    let rankingIntervall: ZaehldatenIntervall[] = [ZaehldatenIntervall.STUNDE_KOMPLETT, ZaehldatenIntervall.STUNDE_HALB, ZaehldatenIntervall.STUNDE_VIERTEL_EINGESCHRAENKT, ZaehldatenIntervall.STUNDE_VIERTEL];
```

Comment:
Direkt in der ZaehldatenIntervall.ts anlegen und dann hier nur noch als const verwenden

Reply:
done